### PR TITLE
Add Gelma's typos that start with "p"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -39086,6 +39096,12 @@ paln->plan, pain, palm,
 paltette->palette
 paltform->platform
 paltforms->platforms
+paly->play, poly, palsy, palmy,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:39099]: Perhaps [`pally`](https://www.oed.com/search/dictionary/?q=pally) too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -39636,11 +39660,16 @@ pasres->parses, pastes, passes, pares,
 pasring->parsing, pasting, passing, paring,
 pass-thru->pass-through, pass through, passthrough,
 pass-trough->pass-through, pass through, passthrough,
+passanger->passenger
+passangers->passengers
 passerbys->passersby
+passess->passes, possess, passers,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:39666]: No typos of `assess` I guess?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -41391,9 +41472,24 @@ predefineds->predefined
...
+predicit->predict
+predicitability->predictability
+predicitable->predictable
+predicitably->predictably
+predicited->predicted
+prediciting->predicting
+predicition->prediction, predication,
+predicitions->predictions, predications,
+predicitng->predicting
+prediciton->prediction
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:41487]: Add typos of `predictive` and `predictor`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -42191,16 +42335,31 @@ profesion->profusion, profession,
...
+professinally->professionally
+professins->professions
 professionaly->professionally
+professionnal->professional
+professionnally->professionally
 proffesed->professed
 proffesion->profession
 proffesional->professional
+proffesionally->professionally
+proffesions->professions
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:42354]: Perhaps add typos of all verbal forms of `profess`, where applicable?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -42924,16 +43100,23 @@ pupulated->populated
...
+purcahsed->purchased
+purcahses->purchases
+purcahsing->purchasing
 purgable->purgeable
 purgest->purges
 puritannical->puritanical
 purposedly->purposely
 purpotedly->purportedly
-purpse->purpose
+purpse->purpose, purse,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43116]: Perhaps `purple`?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:39666]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**